.gitignore

```diff
@@ -2,4 +2,4 @@
 /logs/**
 /node_modules/**
 /public/assets/css/style.css
-/public/assets/lib/bower_components/**
+/public/assets/lib/bower_components/**
```

Comment:
"no new line at end of file".

This file should be removed from this PR if you'll add the newline at the end of the file :-)